static/js/util/profile_edit.js

```diff
@@ -62,13 +63,17 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
         labelStyle={styles.labelStyle}
         value={obj.value}
         label={label}
+        aria-labelledby={labelId}
```

Comment:
Unfortunately I don't think we can do something like:

```js
<label>
  A label
  <RadioButton
    prop={val}
  />
</label>
```

because we render those `RadioButton`s as the children of a `RadioButtonGroup`. `RadioButtonGroup` assumes that all of it's children are `RadioButton` elements at the top level...annoying.

Not sure what to do here. When I set the `label` attribute on `RadioButton` it doesn't render the input inside of the label element OR use `aria-labelledby` to point the screenreader to the appropriate label, at least as far as I can tell. Anyway, tryna figure out what to do here.